website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -265,6 +267,7 @@ createQuery(
 ```graphql
 MetricInput {
   name: String!
+  alias: String!
```

Comment:
@courtneyholcomb added this from @Jstein77 [duplicate PR](https://github.com/dbt-labs/docs.getdbt.com/pull/6777/files#diff-a4e431c95f6eebc16186348ffe9d3245087cfb60303dfcf2af4b2f18d1d0176fR590-R599)